.github/workflows/trivy-scan.yaml

```diff
@@ -0,0 +1,53 @@
+# Uses Trivy to scan every pull request, rejecting those with severe, fixable vulnerabilities.
+name: Trivy
+
+on:
+  pull_request:
+    branches:
+      - REL_4_7
+  push:
+    branches:
+      - REL_4_7
+
+jobs:
+  scan:
+    if: ${{ github.repository == 'CrunchyData/postgres-operator' }}
+
+    permissions:
+      # for github/codeql-action/upload-sarif to upload SARIF results
+      security-events: write 
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Run trivy and log detected and fixed vulnerabilities
+      # This report should match the uploaded code scan report below
+      # and is a convenience/redundant effort for those who prefer to
+      # read logs and/or if anything goes wrong with the upload.
+      - name: Log all detected vulnerabilities
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: fs
+          hide-progress: true
+          ignore-unfixed: true
+          skip-dirs: 'testing'
```

Comment:
⛏️ Might want to add a comment here since this diverges from some of the other uses.